gr00t/experiment/data_config.py

```diff
@@ -674,6 +674,101 @@ def transform(self):
 
 ###########################################################################################
 
+class SinglePandaUSDataConfig(BaseDataConfig):
+    """
+    Data configuration for the single Panda ultrasound dataset. The Franka Panda robot is equipped with a
+    custom ultrasound probe, a wrist camera, and a room camera. The dataset is simulated using 
+    Isaac for healthcare (https://github.com/isaac-for-healthcare). Using `mean_std` normalization for the state and action
+    shows much better performance than `min_max` normalization.
```

Comment:
thanks for experimenting with various normalization scheme. Can we add this short desc to the readme here https://github.com/NVIDIA/Isaac-GR00T/blob/main/getting_started/examples/README.md

Also can add the gif too. 

`